pvt/benchmarks/src/PoolBenchmark.behavior.ts

```diff
@@ -48,12 +48,14 @@ export type TestsAddLiquidityHooks = {
 
 export type TestSettings = {
   offNestedPoolTests: boolean;
```

Comment:
I never noticed this; it's kind of bizarre English. Should be either "turnOffXXXTests" or "XXXTestsDisabled"; something like that.

Reply:
I renamed to disable...Tests, is that ok?